packages/extension/webpack.config.js

```diff
@@ -191,8 +164,6 @@ module.exports = {
     new CopyWebpackPlugin({
       patterns: [{ from: 'public', to: '.' }],
     }),
-    // plugin to enable browser reloading in development mode
-    extensionReloaderPlugin,
```

Comment:
We should open the backlog task to fix reloader down the line.

Reply:
ticket here: https://dailydotdev.atlassian.net/browse/MI-107